shell/theme-color.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
   'log-bg': '#3c444f',
   'light-border': 'rgba(222,222,222,0.5)',
   'light-active': '#6a549e0f', // rgba($primary, .06)
+  'white-90': 'rgba(255,255,255, 0.9)',
+  'white-60': 'rgba(255,255,255, 0.6)',
+  'white-2': 'rgba(255,255,255, 0.02)',
```

Comment:
Use `white-2: rgba(255,255,255, 0.2)`  and  `white-02: rgba(255,255,255, 0.02)` to be consistent with others.